CORBA_DFRS/src/packet/GetBookedFlightCountOperation.java

```diff
@@ -0,0 +1,18 @@
+package packet;
+
+public class GetBookedFlightCountOperation extends OperationParameters {
+	private String recordType;
+	
+	public GetBookedFlightCountOperation(String recordType){
+		this.recordType = recordType;
+	}
+
+	public String getRecordType() {
+		return recordType;
+	}
+
+	public void setRecordType(String recordType) {
+		this.recordType = recordType;
+	}
+	
```

Comment:
I skipped the Builder on this one because it's really short, but I can put it for consistency if you guys want.